sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClient.java

```diff
@@ -30,6 +29,7 @@
 import java.util.UUID;
 import java.util.function.Function;
 
+import static com.azure.ai.formrecognizer.Transforms.getContentType;
```

Comment:
Feels like it shouldn't belong to `Trasnform`?

Reply:
Yeah, could move this to Utility @mssfang ?

Reply:
Will move to Utility class